deribit-api/src/main/java/io/contek/invoker/deribit/api/websocket/market/TradesChannel.java

```diff
@@ -0,0 +1,56 @@
+package io.contek.invoker.deribit.api.websocket.market;
+
+import com.google.common.base.Joiner;
+import io.contek.invoker.deribit.api.websocket.WebSocketChannel;
+import io.contek.invoker.deribit.api.websocket.common.Params;
+import io.contek.invoker.deribit.api.websocket.common.Trade;
+import io.contek.invoker.deribit.api.websocket.common.WebSocketChannelMessage;
+import io.contek.invoker.deribit.api.websocket.common.constants.WebSocketChannelKeys;
+
+import javax.annotation.concurrent.NotThreadSafe;
+import javax.annotation.concurrent.ThreadSafe;
+import java.util.List;
+
+import static io.contek.invoker.deribit.api.websocket.common.constants.WebSocketChannelKeys._trades;
+
+@ThreadSafe
+public final class TradesChannel extends WebSocketChannel<TradesChannel.Message> {
+
+  private final String instrumentName;
```

Comment:
the full channel name should be kept here.
and then we don't have to extract the instrument name when we determine whether we should accept a message.